packages/bits/src/lib/image/image.component.ts

```diff
@@ -116,8 +126,11 @@ export class ImageComponent implements OnInit, AfterViewInit {
         let imageHtml: string = "";
         if (_has(image, "code") && _isString(image.code)) {
             imageHtml = image.code;
+            if(image.name && this.ariaLabel !== image.name) {
+                Promise.resolve().then(_ => this.ariaLabel = image.name);
+            }
         } else {
-            imageHtml = `<img src="${this.image}">`;
+            imageHtml = `<img src="${this.image}" alt="${this.imageAlt}">`;
```

Comment:
Please, add the default `imageAlt`. If the user does not add `imageAlt`, the screen reader will read the image src.

Reply:
I have added empty string as default value for imageAlt.  If the user does not add imageAlt, the screen reader will read only the component role ('image') and aria-label (if it will be added)

Reply:
It looks like we've ended up here with two inputs that are doing the same thing.

I think we can easily throw the `ariaLabel` input to the `alt` attribute depending on the conditionals. We could also remove `aria-label` in case `alt` is used and vise versa to prevent screen readers from reading the same info twice.